doc/interface/binding/multi_component_bi_langmuir_ldf.rst

```diff
@@ -0,0 +1,44 @@
+.. _multi_component_bi_langmuir_ldf_config:
+
+Multi Component Bi-Langmuir LDF
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Group /input/model/unit_XXX/adsorption – ADSORPTION_MODEL = MULTI_COMPONENT_BILANGMUIR_LDF**
+
+
+``IS_KINETIC``
+   Selects kinetic or quasi-stationary adsorption mode: 1 = kinetic, 0 =
+   quasi-stationary. If a single value is given, the mode is set for all
+   bound states. Otherwise, the adsorption mode is set for each bound
+   state separately.
+
+===================  =========================  =========================================
+**Type:** int        **Range:** {0,1}  		**Length:** 1/NTOTALBND
+===================  =========================  =========================================
+
+``MCBLDF_KEQ``
```

Comment:
Since the Multi Component Bi-Langmuir model already has the prefix `MCBL`, it would make sense to change this one to `MCBLLDF`. See also for Multi Component Langmuir.

Reply:
Sure, I will change the parameters name as per your suggestion.